trulens_eval/trulens_eval/feedback/v2/feedback.py

```diff
@@ -170,14 +172,69 @@ class Groundedness(Semantics, WithPrompt):
     )
 
 
+LIKERT_0_3_PROMPT = "0 to 3, where 0 is the lowest score according to the criteria and 3 is the highest possible score"
+BINARY_0_1_PROMPT = "0 or 1, where 0 is lowest and negative (i.e. irrelevant or not grounded) and 1 is highest and positive (relevant, grounded, valid, etc.)"
+
+
+class OutputSpace(str, Enum):
+    """
+    Enum for valid output spaces of scores.
+    """
+    LIKERT_0_3 = "likert-0-3"
+    BINARY = "binary"
+    # note: we are deprecating the 0 to 10 output space in favor of the likert-0-3 or binary output space
+
+
+class EvalCriteria(pydantic.BaseModel):
+    criteria: str
+    output_space: str
+
+    @pydantic.field_validator('output_space')
+    def validate_output_space(cls, v):
+        if v not in [OutputSpace.LIKERT_0_3, OutputSpace.BINARY]:
+            raise ValueError(
+                'output_space must be either "likert-0-3" or "binary"'
+            )
+        return v
+
+    def get_output_scale_prompt(self) -> str:
+        # TODO: verify if this is compatible with the flag higher_is_better directionality
+        if self.output_space == OutputSpace.LIKERT_0_3:
+            return LIKERT_0_3_PROMPT
+        elif self.output_space == OutputSpace.BINARY:
+            return BINARY_0_1_PROMPT
+        else:
+            raise ValueError(
+                'output_space must be either "likert-0-3" or "binary"'
+            )
+
+
+@dataclass
 class ContextRelevance(Relevance, WithPrompt):
-    # openai.qs_relevance
-    # openai.qs_relevance_with_cot_reasons
+    system_prompt: ClassVar[PromptTemplate]
+
+    user_prompt: ClassVar[PromptTemplate]
+    verb_confidence_prompt: ClassVar[PromptTemplate]
+    output_space_prompt: ClassVar[str] = LIKERT_0_3_PROMPT
+
+    criteria: str = """
```

Comment:
how does this work for binary output space?

Reply:
User will have to provide the criteria. 

Open to the idea that we provide with default mapping from output_space to criteria but might get a bit messy to manage across different feedback functions

Reply:
Can we do this using the a variation of the output space prompt?

Reply:
sg - will do in the followup PR. also mentioned this in the PR description

Reply:
Would be useful if you can add a quick mention what specific parameterization is in this v0, and what will be in future versions

Reply:
added in description